Client/Ecosia/Extensions/NumberFormatter+Ecosia.swift

```diff
@@ -0,0 +1,17 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+extension NumberFormatter {
+    static func ecosiaCurrency(withoutEuroSymbol: Bool = false) -> NumberFormatter {
+        let formatter = NumberFormatter()
+        formatter.numberStyle = .currency
+        formatter.currencySymbol = withoutEuroSymbol ? "" : "€"
+        formatter.maximumFractionDigits = 0
+        formatter.usesGroupingSeparator = true
+        formatter.currencyGroupingSeparator = ","
```

Comment:
I would expect that this is Locale specific, not a blocker though

Reply:
You're right, but the Germany Locale then sets it to "." which is not how we have it in Figma, and I do agree it looks better with ",". That, combined with the fact that we only have the info in euros, made me think it would be better to hardcode it like shown in Figma.